src/components/Button/Button.js

```diff
@@ -38,101 +36,14 @@ function Button({
   shadow, // boolean
   ...rest
 }) {
-
```

Comment:
Why was all this removed?  Could add on your own Button component, or perhaps rename this to StyledButton, something like that, and keep the button you made.
See comment below for your implementation of the button.

Reply:
oops reverted to the previous Button.js